### PR TITLE
DOC-6493: Add host options

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/1.6/cenm/config-shell.html

### CODE REVIEW

**Review by tomstark99 — COMMENTED** [review]
I wonder if it is worth adding some sort of visible "optional" option here for setting the host too (since we don't have a signer configuration page for whatever reason)

https://anna.preview.docs.r3.com/en/platform/corda/1.6/cenm/signing-service.html#admin-rpc-interface

File: content/en/platform/corda/1.6/cenm/config-shell.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ title: Shell Configuration Parameters
 The port that the shell ssh daemon should listen on.
 
 
+* **sshHost**:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/config-shell.md:28]: updated

File: content/en/platform/corda/1.6/cenm/config-identity-manager-parameters.md [hunk 2]
```diff
@@ -134,6 +137,8 @@ A list of CRLs hosted by the Identity Manager Service, in addition to the CRLs h
     *(Optional)* Enables verbose logging for the socket layer. Defaults to `false`.
   * **reconnect**:
     *(Optional)* Determines if a client should attempt to reconnect if the connection is dropped. Defaults to `true`.
+  * **host**:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/config-identity-manager-parameters.md:140]: updated

File: content/en/platform/corda/1.6/cenm/config-identity-manager-parameters.md [hunk 3]
```diff
@@ -79,6 +79,9 @@ It is needed as this URL is encoded in certificates issued by the Identity Manag
     * **reconnect**:
     Determines if a client should attempt to reconnect if the connection is dropped.
 
+    * **host**:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/config-identity-manager-parameters.md:82]: updated

File: content/en/platform/corda/1.6/cenm/config-network-map-parameters.md [hunk 4]
```diff
@@ -35,6 +35,8 @@ Details on how the service will communicate with the rest of the CENM deployment
   *(Optional)* Enables verbose logging for the socket layer.
   * **reconnect**:
   Informs whether a client should attempt to reconnect if the connection is dropped.
+  * **host**:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/config-network-map-parameters.md:38]: updated

File: content/en/platform/corda/1.6/cenm/config-network-map-parameters.md [hunk 5]
```diff
@@ -111,6 +113,8 @@ version of Corda that does not support the new PKI structure (arbitrary length c
     *(Optional)* Enables verbose logging for the socket layer. Defaults to `false`.
   * **reconnect**:
     *(Optional)* Determines if a client should attempt to reconnect if the connection is dropped. Defaults to `true`.
+  * **host**:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/config-network-map-parameters.md:116]: updated

**Review by tomstark99 — APPROVED** [review]
LGTM

**Pull request merged**